infra/gcp/clusters/k8s-infra-prow-build-trusted/prow-build-trusted/main.tf

```diff
@@ -0,0 +1,84 @@
+/**
+ * Copyright 2020 The Kubernetes Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/*
+This file defines:
+- Google Project k8s-infra-prow-build-trusted to host the cluster
+- GCP Service Account for prow-build-trusted
+- GKE cluster configuration for prow-build-trusted
+- GKE nodepool configuration for prow-build-trusted
+*/
+
+locals {
+  project_id              = "k8s-infra-prow-build-trusted"
```

Comment:
Can I pick on names?  I like symmetry, so I'd expect to see

`k8s-infra-prow-build` + `k8s-infra-prow-trusted`

or

`k8s-infra-prow-untrusted` + `k8s-infra-prow-trusted`

or

`k8s-infra-prow-build-untrusted` + `k8s-infra-prow-build-trusted`


Is there a reason not to?

Reply:
Well.  I like all of these suggestions better than what I chose.  I prefer the first since the names are shortest.

The reason not to would be that renaming the project id at this point is going to involve creating a new project/cluster/nodepool combo along with the requisite coordination with prow.k8s.io oncall.  The blocker at the moment is our projects being capped at billing quota.

I can file an issue to redo the trusted cluster as `k8s-infra-prow-trusted/prow-trusted` and use it as an opportunity to have someone shadow, or someone else go through this while I watch.

Reply:
If you think the rename is worth it I'll open a ticket, WDYT?

Reply:
Fair.